Makefile

```
@@ -4,12 +4,29 @@ GREEN := $(ESC)[0;32m
 PURPLE := $(ESC)[0;35m
 ZERO := $(ESC)[0;0m
 
+CONFIG_THERMO_NONE  = 0
+CONFIG_THERMO_KTYPE = 1
+CONFIG_THERMO_PT100 = 2
+CONFIG_THERMO_MAX   = 2
+
 CONFIG_THERMO := Ktype
 CONFIG_MAGNETRON  := 0
 CONFIG_HOSTNAME   := "pico_furnace"
 CONFIG_WATER := 1
+CONFIG_THERMO_INTERNAL = $(CONFIG_THERMO_NONE)
+
+ifeq ($(CONFIG_THERMO),Ktype)
+	CONFIG_THERMO_INTERNAL=$(CONFIG_THERMO_KTYPE)
+else ifeq ($(CONFIG_THERMO),pt100)
+	CONFIG_THERMO_INTERNAL=$(CONFIG_THERMO_PT100)
```

Let's error out when user pass invalid value